tests/h/search/client_test.py

```diff
@@ -7,10 +7,28 @@
 from elasticsearch import RequestsHttpConnection
 
 from h.search.client import get_client
+from h.search.client import Client
 
 
 class TestClient(object):
-    pass
+
+    def test_it_sets_the_index_and_conn_properties(self):
+        client = Client(host="http://localhost:9200", index="hypothesis")
+
+        assert client.index == "hypothesis"
```

Comment:
What does this particular assertion test? Given that you've set `index` to "hypothesis" in the autospec creation, is this testing the client behavior or the autospec behavior :) ?

Reply:
This is testing the real class (not mocked)